jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -396,8 +408,12 @@ def vue_do_aper_phot(self, *args, **kwargs):
                     gs = Gaussian1D(amplitude=y_max, mean=0, stddev=std,
                                     fixed={'mean': True, 'amplitude': True},
                                     bounds={'amplitude': (y_max * 0.5, y_max)})
+                    if Version(astropy.__version__) <= Version('5.1'):
+                        fitter_kw = {}
+                    else:
+                        fitter_kw = {'filter_non_finite': True}

```

Comment:
@WilliamJamieson , is this the right way to use https://github.com/astropy/astropy/pull/13259 ?

Reply:
This will work.